services/leafSpaceIntegration/leafspaceconnection/main.go

```diff
@@ -0,0 +1,19 @@
+package leafspaceconnection
+
+import (
+	"fmt"
+	"github.com/leanspace/terraform-provider-leanspace/provider"
+)
+
+var LeafSpaceIntegrationDataType = provider.DataSourceType[LeafSpaceConnection, *LeafSpaceConnection]{
+	ResourceIdentifier: "leanspace_leaf_space_integration",
+	Path:               "integration-leafspace/connections",
+	Schema:             leafSpaceConnectionSchema,
+	FilterSchema:       nil,
+	ReadPath: func(id string) string {
+		return fmt.Sprintf("integration-leafspace/connections")
+	},
+	DeletePath: func(id string) string {
+		return fmt.Sprintf("integration-leafspace/connections")
```

Comment:
From the docs, there is a connectionId: https://docs.leanspace.io/docs/api/branches/dev/leafspace/operations/delete-a-connection

Reply:
Arg it as been modificated https://github.com/leanspace/integration-leafspace-repository/commit/4f054db203a2ff239317b13f4e9a0e65f813565a